webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -603,6 +608,253 @@ private static void addScrambles(PdfWriter docWriter, Document doc, ScrambleRequ
         doc.newPage();
     }
 
+    private static void addScrambleSheet(Document doc, ScrambleRequest scrambleRequest, String globalTitle, String scramble, PdfContentByte cb, int index) throws DocumentException, IOException {
+        Rectangle pageSize = doc.getPageSize();
+
+        BaseFont bf = BaseFont.createFont(BaseFont.HELVETICA, BaseFont.CP1252, BaseFont.NOT_EMBEDDED);
+        BaseFont bfBold = BaseFont.createFont(BaseFont.HELVETICA_BOLD, BaseFont.CP1252, BaseFont.NOT_EMBEDDED);
+
+        int bottom = 30;
+        int left = 35;
+        int right = (int) (pageSize.getWidth()-left);
+        int top = (int) (pageSize.getHeight()-bottom);
+
+        int height = top - bottom;
+        int width = right - left;
+
+
+
+        int fontSize = 15;
+        int padding = 80;
+        int marginBottom = 10;
+        int offsetTop = 27;
+
+        int availableScrambleSpace = width - padding;
+        int scrambleFontSize = 20;
+        float scrambleWidth;
+        do {
+            scrambleFontSize--;
+            scrambleWidth = bf.getWidthPoint(scramble, scrambleFontSize);
+        } while (scrambleWidth > availableScrambleSpace);
+
+        int availableScrambleWidth = (int) (width * .45);
+        int availableScrambleHeight = height - (int) (height * .77) - 90;
+        Dimension dim = scrambleRequest.scrambler.getPreferredSize(availableScrambleWidth - 2, availableScrambleHeight - 2);
+        PdfTemplate tp = cb.createTemplate(dim.width, dim.height);
+        Graphics2D g2 = new PdfGraphics2D(tp, dim.width, dim.height, new DefaultFontMapper());
+
+        try {
+            Svg svg = scrambleRequest.scrambler.drawScramble(scramble, scrambleRequest.colorScheme);
+            drawSvgToGraphics2D(svg, g2, dim);
+        } catch (InvalidScrambleException e) {
+            l.log(Level.INFO, "", e);
+        } finally {
+            g2.dispose();
+        }
+
+        for (int y = 0; y < 8; y++) {
```

Comment:
While it would be cool to resize everything based on this number, it's okay if stuff isn't computed in terms of this number, I just want to name the variable in order to make it more obvious what this loop is doing.